### PR TITLE
Feature/build website

### PR DESCRIPTION
**Pull request by OmarRamoun**
Build Home component
Build Details component
Home component displays several Country components
Details component displays several Region components
Build Header component which displays the country's name in the Details component or "Today's stats" in the Home component
Build Total component which displays the total data
Build SearchBar component which allows the user to filter by country's or region's name
Implement tests for the App UX
Implement custom input hook
Implement redux for state management
Implement fetching from the [Narrativa API](https://covid19tracking.narrativa.com/index_en.html)
Add styling to follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Desktop
+
+![screenshot](./app_screenshot-2.png)
+
+## Built With
+
+## Live Demo
+
+[Live Demo Link](https://dicodiaz-covid-metrics.netlify.app)
```
Comment (bimbolabuari) [inline README.md]: Currently, the deployed app is not working properly, Kindly fix this 👍

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+.bg-custom2 {
+  background-color: #5176be !important;
+}
+
+.bg-custom3 {
+  background-color: #3c5e9f !important;
+}
+
+.cursor-pointer {
+  cursor: pointer !important;
```
Comment (bimbolabuari) [inline src/App.css:28]: [Optional] This is not a CSS best practice, Kindly avoid excessive use of `!important`

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @MrRamoun, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Correct Gitflow
- The README contains an "Acknowledgements" section that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio)

## Required Changes ♻️
- [ ] As required, Kindly record a video presenting the features of the project
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Desktop
+
+![screenshot](./app_screenshot-2.png)
+
+## Built With
+
+## Live video
+
+[Live video Link](https://www.loom.com/share/d8fa5233765e406194b3a7c0adec5d7d)
```
Comment (Whoistolu) [inline README.md]: Great job providing the link to the presentation 👍🏽

File: README.md (hunk 1 above)
Comment (Whoistolu) [reply README.md]: - [x] [REQUIRED] As recommended by the previous reviewer, the Netlify deployment is still not yet successful. Kindly check the log files to see where the deployment has failed. Kindly note that it is a hard requirement to deploy the project.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @MrRamoun 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi @MrRamoun 

Great job with the deployment and fixing the errors!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**